### DISCUSSION TITLE
MaxLFQ column in diann_report Vs diann_maxlfq from diann-rpackage

### DISCUSSION THREAD

**Discussion by abhivij** [vdemichev/DiaNN#502] [Q&A] [answered]
Hi,

I received an output report from DiaNN, provided by a project collaborator. (1.7.12 version was used)

Columns : Genes.Quantity, Genes.Normalised, Genes.MaxLFQ, Genes.MaxLFQ.Unique are present in the sheet but all empty (unlike the sample output file shown in diann-rpackage github)
PG.Quantity, PG.Normalised, Precursor.Quantity, Precursor.Normalised columns are present with values. The corresponding MaxLFQ columns are however not present. 

Could you please help with these questions ?

- Is there some option to show the MaxLFQ values in the output sheet ? 
- I'm able to compute MaxLFQ for protein groups using diann-rpackage as shown in the last example in https://github.com/vdemichev/diann-rpackage. 
  Is this value different from what would have otherwise been computed initially as MaxLFQ column in the output sheet ?

**Comment by CassyGu** [comment]
I've tried to do this before, this value is slightly different from what would have otherwise been computed initially as MaxLFQ column in the output sheet from DIA-NN, My guess is that the DIANN-Rpackage is too old to be updated.

**Reply by abhivij** [reply]
Thanks for the response.
Any ideas on why the output report doesn't contain MaxLFQ column ? Could it be that an old version of DiaNN was used ?

**Comment by vdemichev** [comment, marked as answer]
@abhivij this is very strange, PG.MaxLFQ should be present, unless you disable MaxLFQ. 'Genes' columns might be absent if a FASTA file with genes annotation is not provided.
Yes, the output of diann R package will be different, as it uses a slightly different algorithm from that employed by DIA-NN itself.

**Reply by abhivij** [reply]
Thank you for the response @vdemichev.
From the github readme, it didn't look like there is an option to disable MaxLFQ. Probably by providing a command in 'additional options' ?

**Reply by vdemichev** [reply]
Yes, --no-maxlfq
